client/src/test/java/io/split/engine/experiments/LocalhostSplitChangeFetcherTest.java

```diff
@@ -0,0 +1,25 @@
+package io.split.engine.experiments;
+
+import io.split.client.dtos.Split;
+import io.split.client.dtos.SplitChange;
+import io.split.engine.common.FetchOptions;
+import org.junit.Assert;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.util.List;
+
+public class LocalhostSplitChangeFetcherTest {
+
+    @Test
+    public void testParseSplitChange(){
+        LocalhostSplitChangeFetcher localhostSplitChangeFetcher = new LocalhostSplitChangeFetcher("src/test/resources", "split_init.json");
+        FetchOptions fetchOptions = Mockito.mock(FetchOptions.class);
+
+        SplitChange splitChange = localhostSplitChangeFetcher.fetch(-1L, fetchOptions);
+
+        List<Split> split = splitChange.splits;
+        Assert.assertEquals(7, split.size());
```

Comment:
As an easy verification maybe `till` and `since` can be checked as well?

Reply:
Added.